### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by javierdelapuente**
Applicable spec: <link>

### Overview

Update README according to the new template.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,19 +16,37 @@ access to their HTTP workloads in Kubernetes by easy integration offered via
 For DevOps and SRE teams, providing ingress for charms that support a relation
 to this charm will be possible via a simple `juju relate` command.
 
-## Project and community
+## Get started
+
+Refer to [Nginx Getting Started](https://charmhub.io/nginx-ingress-integrator/docs/getting-started) tutorial for more details.
```
Comment (erinecon) [inline README.md:21]: Nitpick: I would rephrase as `Refer to the [Nginx tutorial](https://charmhub.io/nginx-ingress-integrator/docs/getting-started) for more details on getting started.`

File: README.md [hunk 2]
```diff
@@ -19,19 +16,37 @@ access to their HTTP workloads in Kubernetes by easy integration offered via
...
+  `nginx-route` interfaces.
+- [certificates](https://charmhub.io/nginx-ingress-integrator/integrations#certificates): Requires `tls-certificates`
+  interface that facilitates the use of a TLS certificate.
+
+Refer to [Integrations](https://charmhub.io/nginx-ingress-integrator/integrations/) for more information
+about integrations.
+
+## Learn more
+* [Read more](https://charmhub.io/nginx-ingress-integrator)
+
```
Comment (erinecon) [inline README.md:48]: - Are there any developer docs for Nginx?
- Should we link `https://nginx.org/` as the official website?
- Could you add `[Troubleshooting](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)` as well? (I know the Matrix link appears below, but I think it's good to have it appear in both lists)

Comment (javierdelapuente) [reply README.md:48]: I have added the official and the troubleshooting. I do not think development docs for Nginx are relevant.

**Review by erinecon — APPROVED** [review]
Thanks for updating the README! It looks good to me :)

File: README.md [hunk 3]
```diff
@@ -1,11 +1,8 @@
-[![CharmHub Badge](https://charmhub.io/nginx-ingress-integrator/badge.svg)](https://charmhub.io/nginx-ingress-integrator)
-[![Publish to edge](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/publish_charm.yaml)
-[![Promote charm](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/promote_charm.yaml)
-[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
+# Nginx Ingress Integrator Operator
 
 A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing external access to HTTP/HTTPS services in a
```
Comment (varshigupta12) [inline README.md:3]: I think we should somehow distinguish this as an integrator charm(workload-less) and also how consumer charms could use it. It isn't very clear that this charm doesn't deploy an ingress

File: README.md [hunk 4]
```diff
@@ -19,19 +16,39 @@ access to their HTTP workloads in Kubernetes by easy integration offered via
 For DevOps and SRE teams, providing ingress for charms that support a relation
 to this charm will be possible via a simple `juju relate` command.
 
-## Project and community
+## Get started
+
+Refer to the [Nginx tutorial](https://charmhub.io/nginx-ingress-integrator/docs/getting-started) for more details on getting started.
```
Comment (varshigupta12) [inline README.md:21]: Nit: Maybe remove "Nginx" here?

File: README.md [hunk 5]
```diff
@@ -19,19 +16,39 @@ access to their HTTP workloads in Kubernetes by easy integration offered via
...
 
----
+## Integrations
 
-For further details,
-[see the charm's detailed documentation](https://charmhub.io/nginx-ingress-integrator/docs).
+- [ingress](https://charmhub.io/nginx-ingress-integrator/integrations#ingress): Ingress interface that allows to
+  provide an Ingress for the charms supporting this interface. Charms can get an Ingress using the `ingress` or the
+  `nginx-route` interfaces.
+- [nginx-route](https://charmhub.io/nginx-ingress-integrator/integrations#nginx-route): Ingress interface that allows to
```
Comment (varshigupta12) [inline README.md:37]: Could be good to mention the difference between ingress and nginx-route relations and in which cases to use what

File: README.md [hunk 6]
```diff
@@ -19,19 +16,39 @@ access to their HTTP workloads in Kubernetes by easy integration offered via
...
+  `nginx-route` interfaces.
+- [certificates](https://charmhub.io/nginx-ingress-integrator/integrations#certificates): Requires `tls-certificates`
+  interface that facilitates the use of a TLS certificate.
+
+Refer to [Integrations](https://charmhub.io/nginx-ingress-integrator/integrations/) for more information
+about integrations.
+
+## Learn more
+* [Read more](https://charmhub.io/nginx-ingress-integrator)
+* [Official webpage](https://github.com/nginx)
```
Comment (varshigupta12) [inline README.md:48]: I am not sure if we should add a link to nginx official page here. Maybe change "Official webpage" to "Learn more about Nginx Ingress"?

Comment (javierdelapuente) [reply README.md:48]: I will remove it. 

In reality a probably better reference is the kubernetes ingress https://kubernetes.io/docs/concepts/services-networking/ingress/... but is also difficult to explain...

File: README.md (hunk 5 above)
Comment (javierdelapuente) [reply README.md:37]: I have added a very simple comment of when to prefer one or the other.

File: README.md [hunk 7]
```diff
@@ -19,19 +17,38 @@ access to their HTTP workloads in Kubernetes by easy integration offered via
...
+## Integrations
 
-For further details,
-[see the charm's detailed documentation](https://charmhub.io/nginx-ingress-integrator/docs).
+- [ingress](https://charmhub.io/nginx-ingress-integrator/integrations#ingress): Ingress interface that allows to
+  provide an Ingress for the charms supporting this interface. Charms can get an Ingress using the `ingress` or the
+  `nginx-route` interfaces. Prefer `ingress` if you want to use generic features.
+- [nginx-route](https://charmhub.io/nginx-ingress-integrator/integrations#nginx-route): Ingress interface that allows to
+  provide an Ingress for the charms supporting this interface. Charms can get an Ingress using the `ingress` or the
+  `nginx-route` interfaces. Prefer `nginx-route` if you want to use Nginx specific features.
```
Comment (erinecon) [inline README.md:40]:
```suggestion
  `nginx-route` interfaces. If you want to use Nginx specific features, use `nginx-route`.
```

**Review by erinecon — COMMENTED** [review]
Thanks for updating this README!

**Review by erinecon — APPROVED** [review]

**Review by amandahla — APPROVED** [review]
